public/robots.txt

```diff
@@ -1,2 +1,2 @@
 User-Agent: *
-Disallow: /
+Disallow:
```

Comment:
Is the Disallow empty now / with no value? Shoudnt we not allow access to routes that require a login etc. ?